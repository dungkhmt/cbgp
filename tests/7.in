30 30
30 50
17 10
22 0
5 0
25 13
8 29
16 22
19 0
10 6
15 0
20 3
27 7
11 15
10 12
1 3
16 2
22 3
4 5
19 24
17 1
26 2
17 27
23 0
19 25
22 8
29 14
19 9
21 22
16 18
28 0
12 9
5 2
19 4
13 0
13 4
22 1
23 2
24 2
19 13
19 6
23 15
14 8
22 18
26 16
21 8
19 2
28 19
12 1
16 15
12 14
22 20
1
15 15 35 5 18 5 34 6 14 7 16 7 46 0 43 2 35 2 15 1 45 1 22 7 40 1 12 9 37 2 21 6 24 3 38 8 23 2 3 0 33 7 34 3 28 2 19 6 15 6 12 8 18 4 41 2 0 0 36 3 