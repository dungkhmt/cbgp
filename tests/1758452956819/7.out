(17, 10), (22, 0), (5, 0), (25, 13), (8, 29), (16, 22), (19, 0), (10, 6), (15, 0), (20, 3), (27, 7), (11, 15), (10, 12), (1, 3), (16, 2), (22, 3), (4, 5), (19, 24), (17, 1), (26, 2), (17, 27), (23, 0), (19, 25), (22, 8), (29, 14), (19, 9), (21, 22), (16, 18), (28, 0), (12, 9), (5, 2), (19, 4), (13, 0), (13, 4), (22, 1), (23, 2), (24, 2), (19, 13), (19, 6), (23, 15), (14, 8), (22, 18), (26, 16), (21, 8), (19, 2), (28, 19), (12, 1), (16, 15), (12, 14), (22, 20), 
numberIntersection=0 
minDistance=3.0 
sumDistance=111805.33275942905 
minAngle=0.026779311019765828 
sumAngle=30.996245568118127 
minEdgeLength=3.0 
sumEdgeLength=2353.135098411658 

0.0000000000 3.0000000000 111805.3327594291 0.0267793110 30.9962455681 3.0000000000 2353.1350984117
0: (3, 113),
1: (110, 120),
2: (19, 8),
3: (0, 120),
4: (12, 18),
5: (7, 99),
6: (120, 0),
7: (114, 117),
8: (108, 6),
9: (112, 3),
10: (120, 3),
11: (10, 110),
12: (117, 3),
13: (6, 26),
14: (113, 6),
15: (13, 102),
16: (23, 4),
17: (120, 120),
18: (104, 6),
19: (3, 0),
20: (0, 116),
21: (111, 13),
22: (100, 117),
23: (10, 100),
24: (10, 7),
25: (8, 17),
26: (18, 4),
27: (117, 117),
28: (0, 1),
29: (113, 10),
3 113 110 120 19 8 0 120 12 18 7 99 120 0 114 117 108 6 112 3 120 3 10 110 117 3 6 26 113 6 13 102 23 4 120 120 104 6 3 0 0 116 111 13 100 117 10 100 10 7 8 17 18 4 117 117 0 1 113 10 
0: (3, 113), 1: (110, 120), 2: (19, 8), 3: (0, 120), 4: (12, 18), 5: (7, 99), 6: (120, 0), 7: (114, 117), 8: (108, 6), 9: (112, 3), 10: (120, 3), 11: (10, 110), 12: (117, 3), 13: (6, 26), 14: (113, 6), 15: (13, 102), 16: (23, 4), 17: (120, 120), 18: (104, 6), 19: (3, 0), 20: (0, 116), 21: (111, 13), 22: (100, 117), 23: (10, 100), 24: (10, 7), 25: (8, 17), 26: (18, 4), 27: (117, 117), 28: (0, 1), 29: (113, 10), 
numberIntersection=0 minAngle=0.026779311019765828 sumAngle=30.996245568087744 minEdgeLength=3.0 sumEdgeLength=2353.1350984116643 minDistance=3.0 sumDistance=111805.33275943002 
Time taken: 10.599 s
